lib/tests/lib.rs

```diff
@@ -125,3 +139,112 @@ fn test_builtin_deployed() {
         }
     }
 }
+#[test]
+fn test_constructor_called() {
```

Comment:
This test is redundant with `test_constructor_and_set_up_called`. I think we can remove it.

Reply:
ok, removed it!